test/README.md

```diff
@@ -0,0 +1,8 @@
+# Running tests
```

Comment:
comment not related to this line:
with this PR the "test" folder is owned by shipshape, I would rename to test_shipshape or create a subdirectory /test/shipshape.
So if for any reason we want to create anything "test" related but not shipshape specific, we can do it consistently.

Reply:
@nicob87 thanks for bringing up this point.

The `test` directory was defined as a separate go module, to avoid clashes between test dependencies and skupper dependencies, it is not "owned" by shipshape, but is simply has shipshape as dependency. And it was done this way because shipshape currently depends on an older version of the k8s APIs.

IMHO we should use a standard way for writing our tests, in order to avoid code duplication and promote reuse, and maybe shipshape is not the best way to achieve that (which can be a true and fair statement).

Particularly, I do not like naming a test folder as test_shipshape. Instead I think we can organize tests by level/type inside test directory. Just for the records, inside test directory we currently have `examples/tcpecho` as a separate and individual test suite and we can have other individual suites with different test levels or purposes.

One thing I am also not 100% sure is about using a separate go module for testing (that is one of the reasons for having this PR as a proposal for a testing approach).

If shipshape is not a good fit for writing tests for skupper, or if having a separate go module for testing is a bad practice, then we must discuss other approaches, like creating common base types (like an internal framework) inside this repository to share common needs for all tests we are going to create.

@grs and @ajssmith please share your thoughts as well.

Reply:
@grs @ajssmith @nicob87 I am closing this PR, as per our discussion, it might be better to keep the e2e tests separate from this repo for now, as we are going to start with integration tests here first.
Thanks for the review and suggestions.